cypress/integration/dropdown_spec.js

```diff
@@ -69,8 +69,8 @@ describe('Dropdown Interaction', function() {
     cy.get(DROPDOWN_TOGGLE).should('have.attr', 'aria-expanded', 'false');
 
```

Comment:
As discussed yesterday face-to-face, we should add additional tests in here somewhere to test to make sure it is closing when clicking outside of the menu.